docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1242,6 +1242,49 @@ The behavior is very similar to the <<include_helper>> but:
 
 NOTE: The evaluated template is parsed and evaluated every time the section is executed. In other words, it's not possible to cache the parsed value to conserve resources and optimize the performance.
 
+==== Cached Section
+
+Sometimes it's practical to cache parts of the template that rarely change.
+In order to use the caching capability you'll need to register and configure the built-in `io.quarkus.qute.CacheSectionHelper.Factory`:
```

Comment:
Can `@EngineConfiguration` be used here?

Reply:
Well, you would have to extend the original factory class. Anyway, it should not be needed if the  quarkus-cache extension is used...